### PR TITLE
TipCalc & StarWars Sample update (Android X, MvvmCross 7) 

### PR DESCRIPTION
**Pull request by tuke307**
## Description
Upgraded the complete project from the **StarWarsSample** and the **TipCalc** Sample. 

## Detailled
I installed the latest nuggets, upgraded to MvvmCross 7 and upgraded all android projects (Forms Droid and native droid) to android 10. 
For using AndroidX I had to change a few things like removing the old nuggets(v4/v7), code and view changes.

## Building
All Projects where build fine without any errors. Except the tipcalc.forms.ui.mac project because of this [commit](https://github.com/MvvmCross/MvvmCross-Samples/pull/140). It says the `MainWindow` does not exist. But thats not an error i created.

I tested all the Android projects on an android 11 simulator. I also tested the Windows(WPF,UWP) projects. But i dont have any ios device, so i couldnt test these projects.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross-Samples?pullRequest=153) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross-Samples?pullRequest=153) before we can accept your contribution.<br/><br/>:x: tuke307 [sign now](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross-Samples?pullRequest=153)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/MvvmCross/MvvmCross-Samples?pullRequest=153) it.</sub>

File: StarWarsSample/StarWarsSample.Droid/StarWarsSample.Droid.csproj [hunk 1]
```diff
@@ -33,6 +33,10 @@
     <AotAssemblies>false</AotAssemblies>
     <EnableLLVM>false</EnableLLVM>
     <BundleAssemblies>false</BundleAssemblies>
+    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
+    <AndroidLinkTool>r8</AndroidLinkTool>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
```
Comment (tuke307) [inline StarWarsSample/StarWarsSample.Droid/StarWarsSample.Droid.csproj:39]: needed for deploying androidx

File: StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj [hunk 2]
```diff
@@ -30,6 +30,14 @@
...
     <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
     <AndroidTlsProvider>btls</AndroidTlsProvider>
+    <AotAssemblies>false</AotAssemblies>
+    <EnableLLVM>false</EnableLLVM>
+    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
+    <BundleAssemblies>false</BundleAssemblies>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
+    <AndroidLinkTool>r8</AndroidLinkTool>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidLinkMode>None</AndroidLinkMode>
```
Comment (tuke307) [inline StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj:40]: needed for deploying androidx

File: StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj [hunk 3]
```diff
@@ -51,95 +59,71 @@
...
-    </PackageReference>
-    <PackageReference Include="Xamarin.Android.Support.Fragment">
-      <Version>28.0.0.1</Version>
+      <Version>28.0.0.3</Version>
     </PackageReference>
     <PackageReference Include="Xamarin.Android.Support.Media.Compat">
-      <Version>28.0.0.1</Version>
+      <Version>28.0.0.3</Version>
     </PackageReference>
     <PackageReference Include="Xamarin.Android.Support.v7.AppCompat">
```
Comment (tuke307) [inline StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj:82]: this v7 package should be removed, i forgot it

**Comment by Cheesebaron** [comment]
NICE!

Can you please sign the CLA?

**Comment by PsyhoLord** [comment]
Guys, when you plan to close PR?

**Pull request merged**